src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Sprocs/GetNextTask_3.sql

```diff
@@ -0,0 +1,81 @@
+﻿/*************************************************************
+    Stored procedures for get next available task
+**************************************************************/
+--
+-- STORED PROCEDURE
+--     GetNextTask
+--
+-- DESCRIPTION
+--     Get next available task
+--
+-- PARAMETERS
+--     @queueId
+--         * The ID of the task record
+--     @taskHeartbeatTimeoutThresholdInSeconds
+--         * Timeout threshold in seconds for heart keep alive
+--
+CREATE PROCEDURE [dbo].[GetNextTask_3]
+@queueId VARCHAR (64), @taskHeartbeatTimeoutThresholdInSeconds INT=600
+AS
+
+SET NOCOUNT ON;
+DECLARE @lock VARCHAR(200) = 'GetNextTask_Q='+@queueId
+        ,@taskId VARCHAR (64) = NULL
+        ,@expirationDateTime AS DATETIME2 (7)
+        ,@heartbeatDateTime AS DATETIME2 (7) = SYSUTCDATETIME();
```

Comment:
>@heartbeatDateTime [](http://example.com/codeflow?start=9&length=18)

Change name to StartTime

Reply:
Removed this variable as it is not used.